pom.xml

```diff
@@ -47,56 +47,56 @@
 		<appointmentschedulingVersion>1.7.0</appointmentschedulingVersion>
 		<appointmentschedulinguiVersion>1.9.0-SNAPSHOT</appointmentschedulinguiVersion>
 		<calculationVersion>1.2</calculationVersion>
-		<coreappsVersion>1.24.0-SNAPSHOT</coreappsVersion>
+		<coreappsVersion>1.24.0</coreappsVersion>
 		<databaseBackupVersion>1.3.0</databaseBackupVersion>
 		<dataEntryStatisticsVersion>1.6.0</dataEntryStatisticsVersion>
-		<dataexchangeVersion>1.3.2</dataexchangeVersion>
+		<dataexchangeVersion>1.3.3</dataexchangeVersion>
 		<dataintegrityVersion>4.4.1</dataintegrityVersion>
-		<emrapiVersion>1.27.0</emrapiVersion>
-		<eventVersion>2.6.0</eventVersion>
+		<emrapiVersion>1.28.0-SNAPSHOT</emrapiVersion>
+		<eventVersion>2.7.0</eventVersion>
 		<fhirVersion>1.13.0</fhirVersion>
 		<formentryappVersion>1.4.2</formentryappVersion>
 		<formfilterVersion>1.0.0</formfilterVersion>
-		<htmlformentryuiVersion>1.7.0</htmlformentryuiVersion>
-		<htmlformentryVersion>3.8.0</htmlformentryVersion>
-		<htmlwidgetsVersion>1.9.0</htmlwidgetsVersion>
+		<htmlformentryuiVersion>1.10.0-SNAPSHOT</htmlformentryuiVersion>
+		<htmlformentryVersion>3.9.2</htmlformentryVersion>
+		<htmlwidgetsVersion>1.10.0</htmlwidgetsVersion>
 		<idgenVersion>4.5.0</idgenVersion>
 		<jacksonVersion>1.8.3</jacksonVersion>
 		<junitParamsVersion>1.0.4</junitParamsVersion>
-		<legacyuiVersion>1.4.0</legacyuiVersion>
-		<metadatadeployVersion>1.10.0</metadatadeployVersion>
+		<legacyuiVersion>1.6.0</legacyuiVersion>
+		<metadatadeployVersion>1.11.0</metadatadeployVersion>
 		<metadatamappingVersion>1.3.4</metadatamappingVersion>
 		<metadatasharingVersion>1.5.0</metadatasharingVersion>
 		<openconceptlabVersion>1.1</openconceptlabVersion>
-		<openwebappsVersion>1.9.0</openwebappsVersion>
-		<patientFlagsVersion>3.0.1</patientFlagsVersion>
+		<openwebappsVersion>1.10.0</openwebappsVersion>
+		<patientFlagsVersion>3.0.2</patientFlagsVersion>
 		<providermanagementVersion>2.10.0</providermanagementVersion>
 		<referenceapplicationVersion>2.10.0-SNAPSHOT</referenceapplicationVersion>
-		<referencemetadataVersion>2.6.0</referencemetadataVersion>
+		<referencemetadataVersion>2.10.0</referencemetadataVersion>
 		<registrationappVersion>1.15.0-SNAPSHOT</registrationappVersion>
 		<registrationcoreVersion>1.8.0</registrationcoreVersion>
-		<reportingRestVersion>1.10.0</reportingRestVersion>
-		<reportingVersion>1.16.0</reportingVersion>
+		<reportingRestVersion>1.11.0</reportingRestVersion>
+		<reportingVersion>1.20.0-SNAPSHOT</reportingVersion>
 		<reportingCompatibilityVersion>2.0.6</reportingCompatibilityVersion>
 		<reportingUIVersion>1.6.0</reportingUIVersion>
 		<serializationxstreamVersion>0.2.14</serializationxstreamVersion>
-		<uicommonsVersion>2.9.0-SNAPSHOT</uicommonsVersion>
+		<uicommonsVersion>2.9.0</uicommonsVersion>
 		<uiframeworkVersion>3.15.0</uiframeworkVersion>
 		<uilibraryVersion>2.0.6</uilibraryVersion>
 		<ugandaemrfingerprintVersion>1.0.9-SNAPSHOT</ugandaemrfingerprintVersion>
```

Comment:
update fingerprint to the lates version.

Reply:
@slubwama done